src/autopas/containers/verletClusterLists/ClusterTower.h

```diff
@@ -181,7 +181,13 @@ class ClusterTower : public ParticleCell<Particle> {
    * Returns the number of particles in the tower that are not dummies.
    * @return the number of particles in the tower that are not dummies.
    */
-  [[nodiscard]] size_t getNumActualParticles() const { return _particlesStorage.numParticles() - _numDummyParticles; }
+  [[nodiscard]] size_t getNumActualParticles() const {
+    // This can actually fail in some tests. Also can't do max() because these are size_t
```

Comment:
Why would this fail, shouldn't in these cases numDummyParticles be reset?

Reply:
?

Reply:
I swear I had a test failing because this function returned an underflow but I can not reproduce it anymore. Either I fixed It by changing something else or I'm going crazy :D Either way, I will revert this change so we get a failing test again if `numDummyParticles` is not handled properly.